### PR TITLE
2 corporate sponsorship link

### LINKED ISSUE #2: Add corporate sponsorship information page
Put the document we have on Google Drive that we give to Lincoln organizations up as a page.

### PR DESCRIPTION
**Pull request by troynguyen8**
Added button linking to pdf with sponsorship levels and partnership info. 
Added assets folder to include everything besides css / js

image

closes #2

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -129,40 +129,47 @@ <h5 class="timeline-show-less">Hide Previous Clubs</h5>
...
-              </a>
-              <a href="http://leturaidigima.com" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/partners/letura.png" alt="Letura Idigima">
-              </a>
-              <a href="http://www.carlosvelascodesign.com" target="_blank" rel="noopener">
-                <img class="px-3 logo" src="img/partners/carlos.png" alt="Carlos Velasco Design">
-              </a>
-            </div>
-            </sction>
+          <h3 class="display4 text-center mb-4">Sponsors</h3>
```
Comment (troynguyen8) [inline src/index.html]: I had some mismatched tags in here that I just caught.

previously had <h3></h4> and <section></sction>

**Comment by justinjstark** [comment]
I'd like to take my name and email off the PDF since I don't plan on being involved much in fundraising anymore. We can keep Nicci's email or use a generic sponsor@ email. Can you run this by Nicci and show her the deploy preview to get her feedback?

**Comment by justinjstark** [comment]
Also I just noticed our slogan on the PDF is wrong (it should match our website slogan).

**Comment by troynguyen8** [comment]
@justinjstark got nicci's approval, and changed the PDF as you noted

**Pull request merged**